tests/test_parse_doc.m

```diff
@@ -0,0 +1,19 @@
+function test_parse_doc
```

Comment:
There should be for me other tests. This test coverage is really minimal. When we merge, we should have the certitude that every way to use the function or not use the function is tested and working (specially for parsing functions)

Reply:
I just wanted to draft something here to understand how unit testing works (never did it before). But I totally agree that a maximum of things have to be covered during the test. 👍